app/api/speakers_calls.py

```diff
@@ -36,7 +36,10 @@ def before_create_object(self, data, view_kwargs):
         :return:
         """
         try:
-            self.session.query(SpeakersCall).filter_by(event_id=data['event'], deleted_at=None).one()
+            speakers_call = self.session.query(SpeakersCall).filter_by(event_id=data['event'], deleted_at=None).one()
+            event = speakers_call.event
+            if speakers_call.starts_at >= event.starts_at or speakers_call.ends_at >= event.starts_at:
```

Comment:
Why can't speaker call be at the same time as event start?

Reply:
Included this as well. Please have a look